ci/dlang.yml

```diff
@@ -0,0 +1,20 @@
+name: DLang
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    container:
+      image: dlang2/dmd-ubuntu:latest
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: Build
+      run: dub build

```

Comment:
Having `build` separate is useful as `dub test` ignores the main source file.
However, `dub test` should be first so that the most frequent type of error is caught first.